x-pack/legacy/plugins/security/public/index.scss

```diff
@@ -15,3 +15,6 @@ $secFormWidth: 460px;
 // Public views
 @import './views/index';
 
+// Styles of Kibana Platform plugin
+@import '../../../../plugins/security/public/index';
```

Comment:
__note:__ looks like it's the only way to import style from platform plugin right now.